pkg/apis/admissionregistration/types.go

```diff
@@ -486,14 +508,68 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	ValidationActions []ValidationAction
 }
 
-// ParamRef references a parameter resource
+// ParamRef describes how to locate the params to be used as input to
+// expressions of rules applied by a policy binding.
+//
+// Parameters may be identified by name and selector.
+//
+// If multiple params are found, they are all evaluated with the policy expressions
+// and the results are ANDed together.
+//
+// A single parameter used for all admission requests can be configurated
+// by setting the `name` field, leaving `selector` blank, and setting namespace
+// if `paramKind` is namespace-scoped.
+//
+// A per-namespace parameter may be used by specifying a namespace-scoped
+// `paramKind` in the policy, setting `name` or `selector` and leaving `namespace`
+// in `paramRef` empty.
+//
+// If all fields are left unset, will match ALL resources of the policy's
+// paramKind. If the paramKind is namespace-scoped, will match ALL
+// params in the request object's namespace (or `default` is the request object
```

Comment:
I missed the discussion about using `default` as the implicit namespace for cluster-scoped requests that match the binding. That seems really iffy to me.

If paramKind is namespaced, and this policy/binding will match cluster-scoped objects, I think they need to specify the namespace to use to look up params objects.

Reply:
Changed the doc to match this, changed controller implementation and TestParamRef to test for this.